src/View/Sidebar/SidebarWindow.vala

```diff
@@ -230,6 +230,10 @@ public class Sidebar.SidebarWindow : Gtk.Box, Files.SidebarInterface {
         device_listbox.refresh_info ();
     }
 
+    public void focus () {
```

Comment:
`hides inherited method `Gtk.Widget.focus'. Use the `new' keyword if hiding was intentional`

Reply:
Fixed.  Renamed a couple of functions to match more closely their actual effect.